templates/AP_app/ap-taxpayers.html

```diff
@@ -30,7 +31,7 @@ <h1>
                 <tr>
                     <td>{{taxpayer.business_name}}</td>
                     <td>{{taxpayer.taxpayer_identifier}}</td>
-                    <td>{{taxpayer.taxpayer_state}}</td>
+                    <td>{{taxpayer.get_taxpayer_state_display}}</td>
```

Comment:
Could you explain this change? Thanks

Reply:
this is because "taxpayer.taxpayer_state" shows the db value, so it can't be translated but if i use "get_taxpayer_state_display" the page shows the "verbose_name" of the field